cutekit/project.py

```diff
@@ -1,16 +1,14 @@
 import os
 from typing import Optional
 
-
```

Comment:
Can you keep the 2 blank lines before / after the function declaration ?

Reply:
Done